src/components/ProductImage/ProductImage.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import './ProductImage.scss';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+export const CN = 'product-image';
+
+class ProductImage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedImage: '',
+    };
+  }
+
+  clickHandler = (e) => {
+    const selectedImage = e.currentTarget.dataset.src;
+    // eslint-disable-next-line no-invalid-this
+    this.setState({
+      selectedImage,
+    });
+  }
+
+  renderSmallImages = (selectedImage) => {
+    // eslint-disable-next-line no-invalid-this
+    const { imagesArr } = this.props;
+    return imagesArr.map((el, index) => {
+      if (!index) return null;
```

Comment:
Do I understand correctly, that you don't want to show first item from imagesArr?

Reply:
Yes, because first item goes into big container.